hw/ip/prim/dv/prim_lfsr/data/prim_lfsr_cover.cfg

```diff
@@ -0,0 +1,14 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifer: Apache-2.0
+//
+// The FPV testbench should be considered the verification source of truth
```

Comment:
Please remove this comment as well - it is irrelevant.

Reply:
Done, thanks for the suggestion!